packages/flutter/lib/src/material/chip.dart

```diff
@@ -118,6 +118,11 @@ abstract class ChipAttributes {
   ///
   /// Defaults to 0. The value is always non-negative.
   double get elevation;
+
+  /// Color to be used for the chip's shadow when the elevation is greater than 0.
+  ///
+  /// The default is [Colors.black].
+  Color get shadowColor;
```

Comment:
Get another opinion on whether 'normalShadowColor' would be better.

Reply:
I'm leaning towards just `shadowColor` but will see what @HansMuller thinks.

Reply:
In the recent BottomNavigationBar change and in the TabBar theme we used selectedFoo and unselectedFoo but we haven't been consistent about the naming convention. Using shadowColor and selectedShadowColor is OK with me.